presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestScope.java

```diff
@@ -32,12 +32,12 @@ public void test()
     {
         Scope root = Scope.create();
 
-        Field outerColumn1 = Field.newQualified(QualifiedName.of("outer", "column1"), Optional.of("c1"), BIGINT, false, Optional.empty(), false);
-        Field outerColumn2 = Field.newQualified(QualifiedName.of("outer", "column2"), Optional.of("c2"), BIGINT, false, Optional.empty(), false);
+        Field outerColumn1 = Field.newQualified(QualifiedName.of("outer", "column1"), Optional.of("c1"), BIGINT, false, Optional.empty(), Optional.empty(), false);
```

Comment:
Should we have a test case where this is non-empty?

Reply:
the test is testing scope resolution, and is unrelated to the particular properties of any of the fields